crates/sui-types/src/effects/effects_v1.rs

```diff
@@ -107,6 +107,11 @@ impl TransactionEffectsAPI for TransactionEffectsV1 {
     fn modified_at_versions(&self) -> Vec<(ObjectID, SequenceNumber)> {
         self.modified_at_versions.clone()
     }
+
+    fn lamport_version(&self) -> SequenceNumber {
+        SequenceNumber::lamport_increment(self.modified_at_versions.iter().map(|(_, v)| *v))
```

Comment:
What about receiving objects that don't end up getting received?

Reply:
They still affect the lamport version, don't they? cc @tzakian

Reply:
They do but they may not show up in the effects. 
Oh but wait we decided to ship transfer-to-objects after effects v2 so it can never be in effects v1. So we are probably ok here.